### ISSUE TITLE
Point selection order and labels.

### ISSUE THREAD

**Issue by FernandoRoxo** [pcav/qgis_styles#1]
It seems that the displacement of the dimensioning line is dependent on the order of the point picks, although the label seems to have its own way. In the attached image I picked the upper line clicking on the left point and then finishing on right point.    The lower line I have picked first the right point and finished with the left point.

  I think that if the label were attached in someway to the dimensioning line, this could be an useful feature.

HTH

![dimensioning](https://user-images.githubusercontent.com/36106304/35768077-bff6a786-08dd-11e8-9752-9ef3f9722854.png)

**Comment by pcav** [comment]
Yes, the style could be improved, it's just a quick draft. Please feel free to submit a pull request.
Thanks.
